docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -14,13 +14,58 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.6.1>>
 * <<release-notes-8.6.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.6.1 relnotes
+
```

Comment:
There are some additional items that need to be incorporated. Here is the raw file:

// begin 8.6.1 relnotes

[[release-notes-8.6.1]]
==  8.6.1

Review important information about the  8.6.1 release.





[discrete]
[[known-issues-8.6.1]]
=== Known issues

// TODO: add details and impact

Fleet-Server::

[discrete]
[[known-issue-issue-https://github.com/elastic/elastic-agent/issues/2170]]
.Adding A Fleet Server Integration To An Agent That Was Not Bootstrapped With A Fleet Server Panics
.  {Fleet-Server-issue}https://github.com/elastic/elastic-agent/issues/2170[#https://github.com/elastic/elastic-agent/issues/2170]
[%collapsible]
====

====
osquery::

[discrete]
[[known-issue-issue-https://github.com/elastic/elastic-agent/issues/2066]]
.Osquery Live Query Results Can Take Up To 5 Mins To Show Up In Kibana
. {osquery-pull}https://github.com/elastic/elastic-agent/pull/2076[#https://github.com/elastic/elastic-agent/pull/2076] {osquery-issue}https://github.com/elastic/elastic-agent/issues/2066[#https://github.com/elastic/elastic-agent/issues/2066]
[%collapsible]
====

====








[discrete]
[[bug-fixes-8.6.1]]
=== Bug fixes


elastic-agent::

* Fix Issue Where Beats Started By Agent May Fail With An &#39;Output Unit Has No Config&#39; Error. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2138[#https://github.com/elastic/elastic-agent/pull/2138] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/2086[#https://github.com/elastic/elastic-agent/issues/2086]
* Fix Broken Deployment Using Traffic Filter By Injecting Custom Headers. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2158[#https://github.com/elastic/elastic-agent/pull/2158] {elastic-agent-issue}https://github.com/elastic/beats/issues/32993[#https://github.com/elastic/beats/issues/32993]
* Allow Easily Filtering Only The Agent Logs From The Combined Agent Log Gile. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2044[#https://github.com/elastic/elastic-agent/pull/2044] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/1810[#https://github.com/elastic/elastic-agent/issues/1810]

// end 8.6.1 relnotes